packages/create/src/cli/index.ts

```diff
@@ -0,0 +1 @@
+export { handleCreateFioriCommand } from './create-fiori';
```

Comment:
abstraction question again - would it make sense to move the `create-fiori` code directly in the `index.ts` since (if I understand the structure correctly) will always only export the content from it, and nothing else.

Reply:
I'm not a big fan of having too much code in `index.ts`, but in this case you are correct: intention is to export nothing else than exports of `create-fiori.ts` so I'll move the content.